package/utils/ugps/files/ugps.init

```diff
@@ -12,6 +12,7 @@ service_triggers() {
 
 start_service() {
 	local tty="$(uci get gps.@gps[-1].tty)"
+	local baudrate="$(uci get gps.@gps[-1].baudrate)"
```

Comment:
What if baudrate not defined?

```
local baudrate="$(uci get gps.@gps[-1].baudrate) || echo 0"
```

?

Reply:
Wrong, sh: out of range is appear. See debug with empty baudrate:
```
#!/bin/sh

start_service()
{
        local baudrate="$(uci get gps.@gps[-1].baudrate)"
        [ "$baudrate" -eq 0 ] || echo procd_append_param command "-b ${baudrate}"
}

start_service
```

```
# sh -x ./test1.sh
+ start_service
+ uci get 'gps.@gps[-1].baudrate'
uci: Entry not found
+ local 'baudrate='
+ '['  -eq 0 ]
sh: out of range
+ echo procd_append_param command '-b '
procd_append_param command -b
```

Reply:
oh, makes sense, thanks.